src/probnum/filtsmooth/gaussfiltsmooth/extendedkalman.py

```diff
@@ -67,16 +73,18 @@ def __init__(self, disc_model):
         self.disc_model = disc_model
         super().__init__()
 
-    def transition_realization(self, real, start, **kwargs):
-        return self.disc_model.transition_realization(real, start, **kwargs)
+    def transition_realization(self, real, start, diffusion=1.0, **kwargs):
+        return self.disc_model.transition_realization(
+            real, start, diffusion=diffusion, **kwargs
+        )
 
-    def transition_rv(self, rv, start, linearise_at=None, **kwargs):
+    def transition_rv(self, rv, start, linearise_at=None, diffusion=1.0, **kwargs):
         diffmat = self.disc_model.diffmatfun(start)
         compute_jacobian_at = linearise_at.mean if linearise_at is not None else rv.mean
         jacob = self.disc_model.jacobfun(start, compute_jacobian_at)
         mpred = self.disc_model.dynamicsfun(start, rv.mean)
         crosscov = rv.cov @ jacob.T
-        cpred = jacob @ crosscov + diffmat
+        cpred = jacob @ crosscov + diffusion * diffmat
```

Comment:
This is one of the "leaves" where the parameter is used, not just passed.

Reply:
nice! I am not sure though, whether we want both a diffusion and a diffmat... I find this hard to parse, looking at the code...

Reply:
I did it like this just to touch on the least amount of code. I guess the alternative would be to rename `diffmat` and `diffmatfun` into something else?

Reply:
I agree, both with the fact that it is good to touch the least amount of code, and also with the sentiment that diffmat might be changed instead of diffusion...